### PR TITLE
fix: event when verified

### PR DESCRIPTION
**Pull request by BillySh1**
## Description

Closes # MF-117

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by wenluomask** [comment]
Checked.

File: packages/dashboard/src/pages/Personas/hooks/usePersonaProof.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { useEffect } from 'react'
+import { useAsyncRetry } from 'react-use'
+import { Messages } from '../../../API'
+import { PersonaContext } from './usePersonaContext'
+
+export function usePersonaProof() {
+    const { currentPersona } = PersonaContext.useContainer()
+    const res = useAsyncRetry(async () => {
+        return queryExistedBindingByPersona(currentPersona?.publicHexKey as string)
+    }, [])
```
Comment (Lanttcat) [inline packages/dashboard/src/pages/Personas/hooks/usePersonaProof.ts]: Should depend currentPersona?.publicKey?
And please remove `as string`.

File: packages/shared-base/src/Messages/Mask.ts [hunk 2]
```diff
@@ -58,6 +58,7 @@ export interface MaskEvents extends MaskSettingsEvents, MaskMobileOnlyEvents, Ma
     /** emit when the settings finished syncing with storage. */
     createInternalSettingsUpdated: SettingsUpdateEvent
     ownPersonaChanged: void
+    ownProofChange: void
```
Comment (Lanttcat) [inline packages/shared-base/src/Messages/Mask.ts]: OwnProofChanged?

Comment (BillySh1) [reply packages/shared-base/src/Messages/Mask.ts]: ownProofChanged

File: packages/dashboard/src/pages/Personas/hooks/usePersonaProof.ts (hunk 1 above)
Comment (BillySh1) [reply packages/dashboard/src/pages/Personas/hooks/usePersonaProof.ts]: as string removd,

Comment (BillySh1) [reply packages/dashboard/src/pages/Personas/hooks/usePersonaProof.ts]: refacted

**Comment by wenluomask** [comment]
Checked.

**Review by Lanttcat — APPROVED** [review]

**Pull request merged**